src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index];
+
     }
 }
```

Comment:
The class `ColorSupplier` should have an access modifier. In Java, it's best practice to specify the access level explicitly, even if you want to use the default access level. In this case, it seems like `ColorSupplier` could be public.